neo/io/micromedio.py

```diff
@@ -76,16 +71,16 @@ def read_segment(self, cascade=True, lazy=False, ):
         """
         Arguments:
         """
-        f = StructFile(self.filename, 'rb')
+        f = StructFile(open(self.filename, 'rb'))
```

Comment:
StructFile is a very old wrapper arround struct module of python. And very old code in neo.io was using good old "file()" function. This function disapear with open(). So this is a quite and dirty fix.

For newly coded IOs. It really better to use the numpy.dtype with fields

So NeuroExplorer IO should be rewritten with this in mind.

Note that I am starting a strong refactoring here #354 around neo.io.

Maybe you could have a look there because tthis new neo.rawio could used in mne.